saleor/graphql/tax/mutations/tax_country_configuration_update.py

```diff
@@ -67,14 +68,44 @@ def clean_input(cls, **data):
         cleaned_data = {}
         failed_ids = []
 
+        # Check if only one default rate is provided (only one item without the tax
+        # class).
+        default_rate_items = [
+            item for item in update_tax_class_rates if item.get("tax_class_id") is None
+        ]
+        if len(default_rate_items) > 1:
+            code = (
+                error_codes.TaxCountryConfigurationUpdateErrorCode.ONLY_ONE_DEFAULT_COUNTRY_RATE_ALLOWED  # noqa: E501
```

Comment:
You can do `error_codes.TaxCountryConfigurationUpdateErrorCode.ONLY_ONE_DEFAULT_COUNTRY_RATE_ALLOWED.value` instead of adding `# noqa: E501`.

Reply:
I'm adding `noqa: E501` to disable the line length check in flake8, as I don't see a way to fit it under 88 chars.

Reply:
Ok, I was thinking about different error.